build/package/mongocli_windows_notarize.sh

```diff
@@ -20,5 +20,12 @@ FILE="dist/windows_windows_amd64_v1/bin/mongocli.exe"
 if [[ -f "$FILE" ]]; then
 	echo "notarizing windows binaries"
 	export NOTARY_SIGNING_KEY=$NOTARY_SIGNING_KEY_MONGOCLI
-	go run ./tools/sign -file "$FILE"
+	podman run \
```

Comment:
any reasons to use podman instead of docker here? if we haven't faced issues in the past w/ this, I'm afraid how many flaky failures we would see here.

Reply:
due to licensing, docker is paid so we need to use a free product